_data/students.yml

```diff
@@ -289,4 +289,11 @@
   image: yeezhi.png
   facebook: yeezhi.leow
   twitter: yee_zhi_leow
-  bio: Eat_Sleep_Ball
\ No newline at end of file
+  bio: Eat_Sleep_Ball
+  
+  -name: Kenneth Chua
```

Comment:
There should a space between `-` and `name` , I guess this is the reason the build is failing.